config/taxonomy/layers/catalog/locations.yaml

```diff
@@ -0,0 +1,5 @@
+definitions:
```

Comment:
Should we rename the layers folder to `example` to indicate that this is an example taxonomy?

Reply:
Yes, I think that's better.

Reply:
Done